localstack/services/apigateway/apigateway_listener.py

```diff
@@ -14,6 +14,11 @@
 from localstack.services.kinesis import kinesis_listener
 from localstack.services.generic_proxy import ProxyListener
 
+try:
```

Comment:
Is there a (working) better way to use urllib in python2 and python3? urllib.parse is not backward compatible!